examples/layout.rs

```diff
@@ -0,0 +1,58 @@
+use plotters::prelude::*;
+
+const OUT_FILE_NAME: &'static str = "plotters-doc-data/layout2.png";
+
+fn main() -> Result<(), Box<dyn std::error::Error>> {
+    const W: u32 = 600;
+    const H: u32 = 400;
+    let root = BitMapBackend::new(OUT_FILE_NAME, (W, H)).into_drawing_area();
+    root.fill(&full_palette::WHITE)?;
+
+    let mut chart = ChartLayout::new(&root);
+    chart
+        .set_chart_title_text("Chart Title")?
+        .set_chart_title_style(("serif", 60.).into_font().with_color(&RED))?
+        .set_left_label_text("Ratio of Sides")?
```

Comment:
Also, this seems odd as we also have https://docs.rs/plotters/0.3.1/plotters/chart/struct.MeshStyle.html#method.x_desc which have the similar effect.

Reply:
I guess this should be a broader discussion about how charts should be configured and what belongs to the chart vs. the coordinate system.

The way plotters seems to do it now is to attach labels to the coordinate system and then adjust whether the axes are printed to the left right/etc. Since `ChartLayout` doesn't handle coordinate systems right now, I just went an API that labels things based on position.

I think that maybe the best solution would be to have both. If you set via the coordinate system, it would automatically call the `set_XXX_label_text`. However if a person wanted to set the text without a coordinate system or to override it, they would be free to.

What do you think?